runtime/tests/org.eclipse.core.tests.runtime/src/org/eclipse/core/tests/runtime/jobs/JobTest.java

```diff
@@ -1602,6 +1620,38 @@ protected IStatus run(IProgressMonitor monitor) {
 		group.done();
 	}
 
+	public void testSetProgressGroup_progressGroupIsCanceled_monitorInJobIsCanceled() throws InterruptedException {
+		Job checkMonitorIsCanceled = createJobThatChecksItsOwnMonitor();
```

Comment:
Please reflect in the variable name that it references a job, i.e., `checkMonitorIsCanceledJob` or the like. Otherwise the variable sounds as if it was a method reference.

Reply:
Changed it to `checkMonitorJob` in a055cfab73062f23630a5873c2eb8c4a7ee1f381